pkg/volume/azure_dd/azure_provision.go

```diff
@@ -118,12 +119,13 @@ func (p *azureDiskProvisioner) Provision(selectedNode *v1.Node, allowedTopologie
 		err                        error
 		resourceGroup              string
 
-		zoned             bool
-		zonePresent       bool
-		zonesPresent      bool
-		strZoned          string
-		availabilityZone  string
-		availabilityZones string
+		zoned                    bool
+		zonePresent              bool
```

Comment:
yep, availabilityZone could be merged into availabilityZones. But since `SelectZoneForVolume` has already handles them inside, they are still kept here